docs/sql-migration-guide.md

```diff
@@ -26,6 +26,8 @@ license: |
 
   - In Spark 3.2, `spark.sql.adaptive.enabled` is enabled by default. To restore the behavior before Spark 3.2, you can set `spark.sql.adaptive.enabled` to `false`.
 
+  - In Spark 3.2, the meta-characters `\n` and `\t` are escaped in the `show()` action. In Spark 3.0 or earlier, the two metacharacters are output as it is.
```

Comment:
`In Spark 3.0 or earlier` -> `In Spark 3.1 or earlier`

Reply:
Ur, do we handle this in complex types like `Array/Map/Struct`? Otherwise, we need to narrow down more specific about the benefit.
> the meta-characters `\n` and `\t` are escaped in the `show()` action

Reply:
> In Spark 3.0 or earlier -> In Spark 3.1 or earlier

Ah, my bad...

Reply:
> Ur, do we handle this in complex types like Array/Map/Struct? The current AS-IS PR seems to handle only the top-level string type. If we don't support complex types, we need to narrow down more specific about the benefit.

Yea, nice catch. I'll update this PR for supporting the suggested case. Thanks!